mbed.h

```diff
@@ -16,6 +16,24 @@
 #ifndef MBED_H
 #define MBED_H
 
+#define MBED_LIBRARY_VERSION 151
+
+#if MBED_CONF_RTOS_PRESENT
+// RTOS present, this is valid only for mbed OS 5
+#define MBED_MAJOR_VERSION 5
+#define MBED_MINOR_VERSION 6
+#define MBED_PATCH_VERSION 0
+
+#else
+// mbed 2
+#define MBED_MAJOR_VERSION 2
+#define MBED_MINOR_VERSION 0
+#define MBED_PATCH_VERSION MBED_LIBRARY_VERSION
+#endif
+
+#define MBED_ENCODE_VERSION(major, minor, patch) ((major)*10000 + (minor)*100 + (patch))
+#define MBED_VERSION MBED_ENCODE_VERSION(MBED_MAJOR_VERSION, MBED_MINOR_VERSION, MBED_PATCH_VERSION)        #define MBED_H
```

Comment:
As pinpointed earlier, please rebase interactively to remove this change

Reply:
thank you, rebase done, removed the commit